paddle/operators/activation_op.h

```diff
@@ -95,6 +95,31 @@ struct SigmoidGradFunctor : public BaseActivationFunctor<T> {
   }
 };
 
+// Originally: logsigmoid(x) = -log (1 + exp(-x))
+// For numerical stability: logsigmoid(x) = - (max(-x, 0) + log(exp(-max(-x, 0))
+// + exp(-x - max(-x, 0))))
```

Comment:
maybe we can put the source URL into the comment? (https://hips.seas.harvard.edu/blog/2013/01/09/computing-log-sum-exp/).
For the follow-up maintainers, he may not know the trick, he even cannot check the formula correctness.

Reply:
Thanks, I added more detailed comments. Please let me know if this seems okay now.